javascript/dwa-starter-vue/src/composables/web5.ts

```diff
@@ -0,0 +1,222 @@
+import { Protocol, Record } from '@web5/api/browser'
+
+import { useWeb5Store } from '@/stores/web5'
+import { storeToRefs } from 'pinia'
+
+export enum DateSort {
+  CreatedAscending = 'createdAscending',
+  CreatedDescending = 'createdDescending',
+  PublishedAscending = 'publishedAscending',
+  PublishedDescending = 'publishedDescending'
+}
+
+export function useWeb5() {
+  const { web5 } = storeToRefs(useWeb5Store())
+
+  const protocol = 'https://didcomm.org/dwa-starter-vue'
```

Comment:
since they are both framework agnostic, can you please maintain the original files below and connect with this composable?
- https://github.com/TBD54566975/tbd-examples/blob/main/javascript/dwa-starter-react-vite/src/lib/todo-dwn-repository.ts
- https://github.com/TBD54566975/tbd-examples/blob/main/javascript/dwa-starter-react-vite/src/web5/protocols.ts

this will make things simpler to maintain across all the dwa-starter examples

Reply:
done with this @leordev, I found the react implementation of the web5 composable a bit confusing hence why I opted for a different version

Reply:
What I suggested was to copy these two files to your project and use them in the `useWeb5` or in the best way you can think of. Is it possible?

Reply:
ah ok got that now

Reply:
should be ok now i hope @leordev